jsonHomeWork.playground/Sources/NetworkMethod.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+//MARK: - MethodCreateUrlRequest
+
+public func getData(urlRequest: String) {
+    let urlRequest = URL(string: urlRequest)
+    guard let url = urlRequest else { return }
```

Comment:
- проще на вход принимать сразу `url`
- вместо просто `return` можно вывести хотя бы в консоль информацию о том, что произошло

```suggestion
public func getData(urlRequest: URL?) {
    guard let url = urlRequest else {
        print("Отсутствует ссылка")
        return
    }
```